packages/devtools-components/src/tree.css

```diff
@@ -36,25 +26,35 @@
   display: block;
 }
 
+.tree-indent {
+  display: inline-block;
+  width: 12px;
+  margin-inline-start: 5px;
+  border-inline-start: 1px solid #A2D1FF;
+}
+
 .tree .tree-node[data-expandable="true"] {
   cursor: default;
 }
 
 .tree .tree-node:not(.focused):hover {
-  background-color: var(--tree-node-hover-background-color);
```

Comment:
Why not using a variable?

Reply:
for consistency sake. I nuked all the Component specific css variables, since the style can style be overidden by adding more specific selectors when needed.